app/views/pages/terms_and_conditions.html.haml

```diff
@@ -0,0 +1,122 @@
+
+        %h2.govuk-heading-m Using GOV.UK Registers
+        %p You agree to use GOV.UK Registers only for lawful purposes. You must also use it in a way that does not infringe the rights of, or restrict or inhibit the use and enjoyment of, this site by anyone else.
+        %p We update GOV.UK Registers all the time. We can change or remove content at any time without notice.
+
+        %h2.govuk-heading-m Dataset services
+        %p You can use GOV.UK Registers to access datasets that can be managed by GDS or another government department or agency.
+
+        %h2.govuk-heading-m Linking to GOV.UK Registers
+        %p We welcome and encourage other websites to link to GOV.UK Registers.
+        %p You must #{link_to 'contact us', support_path} for permission if you want to either:
+        %ul.govuk-list.govuk-list--bullet
+          %li charge your website’s users to click on a link to any page on GOV.UK Registers
+          %li say your website is associated with or endorsed by GOV.UK Registers or another government department or agency
+
+        %h2.govuk-heading-m Using GOV.UK Registers content
+        %p Most content on GOV.UK Registers is subject to #{link_to 'Crown copyright protection', 'https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/'} and is published under the Open Government Licence (OGL).
+        %p Some content is exempt from the OGL - check the #{link_to 'list of exemptions', 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/1/open-government-licence.htm'}.
+        %p Departmental logos and crests are also exempt from the OGL, except when they form an integral part of a document or dataset.
+        %p If any content is not subject to Crown copyright protection or published under the OGL, we’ll usually credit the author or copyright holder.
+        %p You can reproduce content published on GOV.UK Registers under the OGL as long as you follow the licence’s conditions.
+        %p #{link_to 'Contact us', support_path} if you want to reproduce a piece of content but are not sure if it’s covered by Crown copyright or the OGL.
+        %p We make most of the data on GOV.UK Registers available through downloads or APIs for other websites and applications to use. The websites and applications that use our data are not our products, and they might use versions of our data that have been edited and stored for later use (‘cached’).
+        %p We do not give any guarantees, conditions or warranties about the accuracy or completeness of any dataset we do not own. We’re not liable for any loss or damage that may come from your use of these datasets.
+        %p The most up-to-date version of the datasets we show will always be on the original register.
```

Comment:
this doesn't seem accurate to me given that registers can be derived from other data sources